ipns.go

```diff
@@ -24,16 +24,33 @@ func (s *Shell) Publish(node string, value string) error {
 	return nil
 }
 
-// Resolve gets resolves the string provided to an /ipfs/[hash]. If asked to
+// Resolve resolves the string provided to an /ipfs/[hash]. If asked to
 // resolve an empty string, resolve instead resolves the node's own /ipns value.
 func (s *Shell) Resolve(id string) (string, error) {
-	var resp *Response
-	var err error
+	return s.resolve(id, false)
+}
+
+// ResolveFresh resolves the string provided to an /ipfs/[hash] without looking
```

Comment:
it resolves the thing provided (which may be an ipns or an ipfs thing?) to an ipfs hash